### PR TITLE
Fix $tags type

### PR DESCRIPTION
**Pull request by devojifr**
$tags using in the documentation must be a Collection to be persisted.

### CODE REVIEW

**Comment by HeahDude** [comment]
👍 I think we need the same fix line 20.

**Comment by devojifr** [comment]
Exactly!
I've added a new commit.

**Review by HeahDude — APPROVED** [review]
Thank you for that PR!

File: form/form_collections.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ Let's start by creating a ``Task`` entity::
     class Task
     {
         protected string $description;
-        protected ArrayCollection $tags;
+        protected Collection $tags;
```
Comment (OskarStark) [inline form/form_collections.rst:20]: ArrayCollection use statement can be removed

**Comment by javiereguiluz** [comment]
@devojifr thanks a lot for improving this ... and congrats on your first Symfony Docs contribution 🎉

File: form/form_collections.rst (hunk 1 above)
Comment (javiereguiluz) [reply form/form_collections.rst:20]: I did this while merging. Thanks.

**Pull request merged**